### ISSUE TITLE
Tutorial: Self-Supervised Pretraining Pipeline | Fine-Tuning on External dataset to show the benefit (ETA 11/12/2021)

### ISSUE THREAD

**Issue by finalelement** [Project-MONAI/MONAI#2708]
Self-supervised based pretraining pipeline using augmentation on the image which will patch shuffle, use noise, flips and in-cuts and out cuts. The network will try to reconstruct the original image

**Comment by finalelement** [comment]
The tutorial is coming along, the experimental results have worked out well. With pretraining we can show improvements upto 7% given small finetuning data set size of 6 training volumes with a validation of 6 volumes on Beyond the cranial vault MICCAI 2015 challenge

**Comment by finalelement** [comment]
The tutorial has been completed, please check it out [here](https://github.com/Project-MONAI/tutorials/tree/master/self_supervised_pretraining)

There maybe minor edits in readme, nothing that will impact the overall results for it.
